Lib/test/test_buffer.py

```diff
@@ -4412,6 +4417,13 @@ def test_issue_7385(self):
         x = ndarray([1,2,3], shape=[3], flags=ND_GETBUF_FAIL)
         self.assertRaises(BufferError, memoryview, x)
 
+    @support.cpython_only
```

Comment:
@support.cpython_only should be removed: it's redundant with @unittest.skipUnless(_testcapi, 'requires _testcapi').